_docs_agent/host-feature.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: HOST feature
+description: Monitor services that run on a host from our Docker plugin
+---
+
+If the CoScale agent is installed as a Container, you can now gather data from services running on the host, without creating and installing another agent as a package.
```

Comment:
Small update:

If the CoScale agent is installed as a Container, you can gather data from services running on the host, without installing an agent as a package on the host. This mechanism can be used to parse log files or execute a script on the host instead of a container running on the host. It is typically used to monitor services that are running on the host but not in a container.